### PR TITLE
Invisible and minuscule typo in fraud

### PR DESCRIPTION
**Pull request by jdemonasterio**
@patriciorocca This typo was displayed in the blog; amazingly surpassing @GervasioCalderon 's filter 😛 

All else is fine. LLG

### CODE REVIEW

**Comment by GervasioCalderon** [comment]
@jdemonasterio : Puede fallar (no translation available for Tusam).

**Comment by patriciorocca** [comment]
LGTM

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown [hunk 1]
```diff
@@ -101,7 +101,7 @@ Finally, to build an install threshold for today's date, we take a timespan of d
...
+The methods here exposed are a first iteration for fraud detection and classification for Click Spamming. Note here that we rely on the assumption that for all apps there are periods free of fraudulent behavior. These periods are then used to calculate our final threshold which is robust to data that is contaminated. We are confident on this assumption since we have seen that fraudulent behavior from publishers will only last, at most, for a few days. Thus using at least a week of data is enough to identify cases of fraudulent behavior.
 
 We find that this algorithm is strong and flexible to account for differences among applications, where there are significant time differences between TimeDeltas. The evaluation measures this difference by automatically fitting the best distributions and usesworks which uses it to provide a robust and thorough fraud detection system.
```
Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: I don't like the repetition in "usesworks which uses it", too cacophonic for my ears. And what's a useswork anyway?

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown [hunk 2]
```diff
@@ -101,7 +101,7 @@ Finally, to build an install threshold for today's date, we take a timespan of d
 
 #Conclusion
 
-The methods here exposed are a first iteration for fraud detection and classification for Click Spamming. Note here that we rely on the assumption that for all apps, there are periods free of fraudulent behavior. These periods are then used to calculate our final threshold which is robust to data that is contaminated. We are confident on this assumption since we have seen that fraudulent behavior from publishers will only last, at most, for a few days. Thus using at least a week of data is enough to identify cases of fraudulent behavior.
+The methods here exposed are a first iteration for fraud detection and classification for Click Spamming. Note here that we rely on the assumption that for all apps there are periods free of fraudulent behavior. These periods are then used to calculate our final threshold which is robust to data that is contaminated. We are confident on this assumption since we have seen that fraudulent behavior from publishers will only last, at most, for a few days. Thus using at least a week of data is enough to identify cases of fraudulent behavior.
```
Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: "at least a week of data" -> "at least one week of data" makes more emphasis in that "week" is an amount.

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown [hunk 3]
```diff
@@ -71,26 +71,26 @@ $$KL(Q \mid P) =  -\int p(x) ln(\frac{q(x)}{p(x)})dx$$
...
 4. Get the 95th theoretical percentile as our threshold, if the best-fit distribution is non-fraudulent.
 
-Here, we must remind readers that we have chosen weak theoretical distributions to fit the fraudulent cases. This means that we are imposing a higher barrier to the fraudulent case to be selected. This is because we are lowering the amount of false positives cases and because this will give us more certainty on our classification.
+Here, we must remind readers that we have chosen weak theoretical distributions to fit the fraudulent cases. This means that we are imposing a higher barrier to the fraudulent case to be selected. This is because we are lowering the amount of false positives cases and also this will give us more certainty on our classification.
 
-Finally, we'll change the timeframe of analysis and repeat the algorithm before to output for given time period, a TimeDelta threshold and a a classification of fraudulent vs. non-fraudulent behavior.
+Finally, we'll change the timeframe of analysis and repeat the algorithm before to output for a given time period, a TimeDelta threshold and a a classification of fraudulent vs. non-fraudulent behavior.
```
Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: and a a classification -> and a classification

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown (hunk 2 above)
Comment (jdemonasterio) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: We can endlessly revise the nuances of the English language. I think that we can start wrapping up if you guys approve.

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown [hunk 4]
```diff
@@ -71,26 +71,26 @@ $$KL(Q \mid P) =  -\int p(x) ln(\frac{q(x)}{p(x)})dx$$
 
 The second form better characterizes how the $Q$ distribution is used as an approximation of $P$ by comparing the entropy $H(P)$, with the entropy when we use $Q$ as an approximation.
 
-For our specific case we will comparing the theoretical distributions `P` with the empirical distribution `Q`. The idea is to try and decide in which category does the empirical distribution fall to. And then use the theoretically fit distribution to set a 95th percentile threshold on the data. This _ad-hoc_ percentile will serve as a cut to all late timed installs. 
+For our specific case we will comparing the theoretical distributions `P` with the empirical distribution `Q`. The idea is to try and decide which category does the empirical distribution fall to. And then use the theoretically fit distribution to set the 95th percentile threshold on the data. This _ad-hoc_ percentile will serve as a cut to all late timed installs.
```
Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: and decide which category does the empirical distribution fall to -> and decide which category the empirical distribution falls to

Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: For our specific case we will comparing the theoretical distributions -> For our specific case we will compare the theoretical distributions (or will be comparing, but will compare sounds more specific in time).

Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: the theoretical distributions `P` -> the theoretical distribution `P`

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown (hunk 2 above)
Comment (GervasioCalderon) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: Ok, this one is subtle, but not the ones below, such as " a a ".

**Pull request merged**